contracts/LockedDealV2.sol

```diff
@@ -25,6 +25,7 @@ contract LockedDealV2 is LockedPoolzData {
     //@dev no use of revert to make sure the loop will work
     function WithdrawToken(uint256 _PoolId)
         external
+        isLocked(_PoolId)
```

Comment:
This withdraw function may be used by another contract in a loop, hence we do not want this to fail. `isLocked` Modifier is not needed at all. Let modify `getWithdrawableAmount` method to return `0` before the `CliffTime`.